src/tufw/__init__.py

```diff
@@ -4,7 +4,7 @@
 from sys import argv, executable
 
 from dialog import Dialog
-from firewall import *
+from tufw.firewall import *
```

Comment:
```suggestion
from .firewall import *
```

Reply:
@SeRj-ThuramS can you approve this fix. It works for me with python 3.8.10 on ubuntu with the `.firewall` relative import